### ISSUE TITLE
Picture cover post

### ISSUE THREAD

**Issue by MiFrance** [giuscris/formwork#18]
Have a issue with cover image post.
If I select an image, it is not displayed on the blog or on the article concerned.

**Comment by giuscris** [comment]
It's a templates issue. Fix it soon, thanks for discovering it 👍

**Comment by giuscris** [comment]
Fixed!

**Comment by MiFrance** [comment]
Cover don't work for me on MAMP with 2 template change...
I added cover image on 2 blog post and don't show.

**Comment by giuscris** [comment]
Could you post the content of the affected pages?

**Comment by MiFrance** [comment]
```
---
title: Hello World!
publish-date: 2018-06-15
image: danger_filtrage.jpg
---
This is an example of blog post.
```

**Comment by giuscris** [comment]
Where is the image located? The filename is relative to the page.

**Comment by MiFrance** [comment]
<img width="1105" alt="Capture d’écran 2019-03-18 à 00 20 02" src="https://user-images.githubusercontent.com/12760053/54499576-9d133a00-4913-11e9-92aa-d101ee3d5469.png">

**Comment by MiFrance** [comment]
And code of templates/partials/cover-image.php
```
<?php if ($page->has('image')): ?>
    <div class="cover-image" style="background-image:url(<?= $page->get('image') ?>);"></div>
<?php endif; ?>
```

**Comment by giuscris** [comment]
Can you see the image at `http://path/to/page/danger_filtrage.jpg`?

**Comment by MiFrance** [comment]
If i add a picture in post OR page is't OK
Is just the cover image don't show

**Comment by MiFrance** [comment]
Example to work:
```
---
title: Hello World!
publish-date: 2018-06-15
image: danger_filtrage.jpg
---
This is an example of blog post.

![](danger_filtrage.jpg)
```

**Comment by giuscris** [comment]
> If i add a picture in post OR page is't OK
> Is just the cover image don't show

It seems the image is correctly served so the problem shouldn't be in the backend. Have you tried with another image?

**Comment by MiFrance** [comment]
Yes same issue with another image in other posts...
Is just for the cover image.

**Comment by giuscris** [comment]
![image](https://user-images.githubusercontent.com/18699708/54499818-4ce9a700-4916-11e9-9cc7-3efe5fc506b0.png)

You should get something like this.

**Comment by giuscris** [comment]
Which browser do you use? I've just discovered that Firefox doesn't render the cover image because of the template styles. I'll fix it soon.

**Comment by MiFrance** [comment]
Exactly ! I use Firefox 65.0.1 for Mac.
I just tryed on Safari and is work great ;)

**Comment by MiFrance** [comment]
### Fix:
In templates/partials/cover-image.php, remplace PHP by:
```
<?php if ($page->has('image')): ?>
    <img class="cover-image" src="<?= $page->get('image') ?>" alt="cover image"/>
<?php endif; ?>
```

**Comment by giuscris** [comment]
You're actually using an `<img>` tag but there are issues with different image sizes, so using the `background-image` is the best approach. Just let me fix issue #21.
